crates/astria-conductor/src/executor/mod.rs

```diff
@@ -254,6 +255,7 @@ impl Executor {
             sequencer_block_hash = ?block.block_hash,
             sequencer_block_height = block.header.height.value(),
             execution_block_hash = hex::encode(&executed_block.hash),
+            tx_count,
```

Comment:
This is getting messy. Can you instrument `execute_block` and move the tx count and the other 2 fields derived from the block into the span?